client/js/_author/components/assessments/question_types/question_common/header/default.jsx

```diff
@@ -1,36 +1,41 @@
-import React    from 'react';
+import React     from 'react';
+import DotLoader from '../../../../common/dot_loader';
 
 export default function DefaultHeader(props) {
-  return (
-    <div className="au-o-right au-c-question-icons">
-      <button
-        className="au-c-btn au-c-btn--square"
-        tabIndex="0"
-        onClick={props.togglePreview}
-      >
-        <i className="material-icons">remove_red_eye</i>
-      </button>
-      <button
-        className="au-c-btn au-c-btn--square"
-        tabIndex="0"
-        onClick={props.toggleReorder}
-      >
-        <i className="material-icons">swap_vert</i>
-      </button>
-      <button
-        className="au-c-btn au-c-btn--square"
-        tabIndex="0"
-        onClick={() => props.deleteAssessmentItem(props.id)}
-      >
-        <i className="material-icons">delete</i>
-      </button>
-    </div>
-  );
+  if (!props.isRemoving) {
```

Comment:
Turn this into `if (props.isRemoving) return <DotLoader />` instead of wrapping the big thing in an if